src/ddmd/aggregate.h

```diff
@@ -247,6 +247,18 @@ struct ClassFlags
     };
 };
 
+enum ClassKind
+{
+    /// the class is a d(efault) class
+    d,
+    /// the class is a C++ interface
+    cpp,
+    /// the class is an Objective-C class/interface
+    objc,
+    /// the class is anonymous
+    anonymous,
+};
```

Comment:
Won't this be an `int`, which will not match up with the size of `ubyte` on the D side?

Reply:
enums in C++ are ints. Not sure what the fix should be here. Maybe declare an ClassKind as an unsigned char?